docs/_includes/tests/filter-menu-button.html

```diff
@@ -0,0 +1,650 @@
+<h2>Collapsed</h2>
+<div class="demo">
+    <div class="demo__inner">
+        <div class="filter-group">
+            <span class="filter-menu-button">
+                <button type="button" class="filter-menu-button__button">
+                    <span class="filter-menu-button__button-cell">
+                        <span class="filter-menu-button__button-text">Filter Menu Button</span>
+                        <svg class="icon icon--chevron-down" focusable="false" height="12" width="12">
```

Comment:
@seangates @msendlakowski We must also support the background svg use case for the button, e.g. `<span class="filter-menu-button__icon"><span>' instead of <svg> tag.

Reply:
We do. I just did not provide an example of it.

Reply:
I tried it and it did not work.